code/game/objects/items/weapons/autolathe_disks.dm

```diff
@@ -310,7 +310,7 @@
 	..()
 
 /obj/item/weapon/disk/autolathe_disk/nonlethal_ammo
-	name = "Frozen Star Nonlethal Ammo Pack"
```

Comment:
I think Nonlethal all as one word is fine here.
This is an item name, which means space is at a premium